src/auslib/services/releases.py

```diff
@@ -244,6 +244,14 @@ def get_assets(name, trans):
     return assets
 
 
+def get_release_names(trans, product=None):
```

Comment:
Hm, do we want to cache the result? I imagine the answer might be "no" if the DB changed, and "maybe" or "yes" if the DB hasn't, depending on how expensive it is.

Reply:
I thought about this, but I think ultimately it won't win much, if anything. We have so many webheads (and they recycle) and so few requests (once per nightly decision task AFAICT), that the hit rate will be zero, or close enough to it.